### PR TITLE
Add counter to "Tasks" link under "Places"

### LINKED ISSUE #11904: [webui] two Links to tasks
Screenshot from 2021-11-26 13-26-30

There are two 'tasks' buttons, one in the left panel, one in the top panel.

Both link to /my/tasks

### PR DESCRIPTION
**Pull request by dmarcoux**
Also hide the link if the notifications_redesign is disabled.

Closes #11904

### CODE REVIEW

File: src/api/app/views/layouts/webui/_places.html.haml [hunk 1]
```diff
@@ -8,10 +8,16 @@
         = link_to(user_path(User.session!), id: 'link-to-user-home', class: 'nav-link') do
           = image_tag_for(User.session, size: 23, custom_class: 'rounded-circle bg-light mr-2')
           Profile
-    %li.nav-item
-      = link_to(my_tasks_path, class: 'nav-link', title: 'Tasks') do
-        %i.fas.fa-tasks.fa-lg.mr-2
-        %span.nav-item-name Tasks
+    -# Without this, we would have two links to the tasks, one here and one in the top navigation.
+       This is going to be resolved once the notifications_redesign is rolled out.
```
Comment (eduardoj) [inline src/api/app/views/layouts/webui/_places.html.haml:12]:
```suggestion
    -# This is going to be resolved once the notifications_redesign is rolled out.
```

Comment (saraycp) [reply src/api/app/views/layouts/webui/_places.html.haml:12]: An alternative for multiline comments is:

```
/
  line 1
  line 2
```

or 

```
/ -------------------------------------------
/ Comments
/ -------------------------------------------
```

According to this: https://learnxinyminutes.com/docs/haml/

Comment (saraycp) [reply src/api/app/views/layouts/webui/_places.html.haml:12]: Both pass linters for me.

Comment (dmarcoux) [reply src/api/app/views/layouts/webui/_places.html.haml:12]: Sure, but this is already working, so I don't see why it's needed to change. The text `This is going to be resolved once the notifications_redesign is rolled out.` isn't displayed in the view, so as expected, it's processed as a comment.

**Review by saraycp — APPROVED** [review]

File: src/api/app/views/layouts/webui/_places.html.haml (hunk 1 above)
Comment (saraycp) [reply src/api/app/views/layouts/webui/_places.html.haml:12]: This is just a comment, not a change request. Feel free to add it or not.

Comment (saraycp) [reply src/api/app/views/layouts/webui/_places.html.haml:12]: If I had to choose, I would use:

```
/ -------------------------------------------
/ Comments
/ -------------------------------------------
```

**Pull request merged**